src/app/detail/ui/TaskEditor.tsx

```diff
@@ -17,7 +17,8 @@ import { SupabaseActions } from '@/utils/SupabaseActions'
 import { generateRandomString } from '@/utils/generateRandomString'
 import { ISignedUrlUpload, UserType } from '@/types/interfaces'
 import { WorkflowStateResponse } from '@/types/dto/workflowStates.dto'
-import { TapWriteTaskEditor } from '@/app/detail/ui/styledComponent'
+import { Tapwrite } from 'tapwrite'
+import '@/app/tapwrite.css'
```

Comment:
We need `tapwrite.css` for the task description in the `NewTaskForm` as well which is in the root. Do you think importing this here will affect in `NewTaskForm`? If not then it would be better to apply this from the root.